sympy/stats/tests/test_continuous_rv.py

```diff
@@ -1226,7 +1235,22 @@ def test_NormalDistribution():
     assert nd.expectation(1, x) == 1
     assert nd.expectation(x, x) == 0
     assert nd.expectation(x**2, x) == 1
-
+    #Test issue 10076
+    a = SingleContinuousPSpace(x, NormalDistribution(2, 4))
+    _z = Dummy('_z')
+    expected1 = Integral(sqrt(2)*exp(-(_z - 2)**2/32)/(8*sqrt(pi)),
+                (_z, -oo, 1))
+    assert (a.probability(x < 1, evaluate=False)).dummy_eq(expected1) is True
+    expected2 = Integral(sqrt(2)*exp(-(_z - 2)**2/32)/(8*sqrt(pi)),
+                (_z, 1, oo))
```

Comment:
These tests are hard to read. Perhaps at least a blank line between each one would help. Also is it necessary to move the integral limits to the line below?

Reply:
I have moved them down to wrap up the line within 79 characters, Would move them to single line and a blank space.